docs/language-server/protocol-language-server.md

```diff
@@ -1530,17 +1530,17 @@ The component group provided by a library.
 ```typescript
 interface LibraryComponentGroup {
   /**
-   * Thf fully qualified module name. A string consisting of a namespace and
+   * The fully qualified library name. A string consisting of a namespace and
    * a library name separated by the dot <namespace>.<library name>,
    * i.e. `Standard.Base`.
    */
   library: string;
```

Comment:
Note: I kept the name of the `library` field without change, for consistency with other messages in the Language Server protocol - for example, compare with the field names in the `MethodPointer` message:

```
interface MethodPointer {
  /** The fully qualified module name. */
  module: String;

  /** The type on which the method is defined. */
  definedOnType: String;

  /** The method name. */
  name: String;
}
```